### PR TITLE
TLUNIZIN-681 Create /status URLs for Canvas Course Manager

### PR DESCRIPTION
**Pull request by dovek**
Due: 11/9
@dlhaines 
@gsilver

### CODE REVIEW

File: src/main/webapp/status.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en" ng-app="sectionsApp">
+
+<head>
+<meta http-equiv="Content-Type" content="text/html; Charset=UTF-8">
+<title>Canvas Course Manager Status Page</title>
+<link href="bootstrap/css/bootstrap.css" type="text/css"
```
Comment (gsilver) [inline src/main/webapp/status.html]: This URL does not resolve - try:
<link href="vendor/bootstrap/bootstrap.min.css" type="text/css"
	rel="stylesheet" media="all">

File: src/main/webapp/status.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+			Status Ping: <a href="status/ping.html">Here</a>
+		</div>
+
+	</div>
+
+	<h1>Points of Integration</h1>
+
+	<div>
+		<div id="dev-canvas">
+			<b>Canvas</b>: The Canvas API is used by this application to make
```
Comment (gsilver) [inline src/main/webapp/status.html]: <b> should be <strong>

File: src/main/webapp/status.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+		<div id="statusPing">
+			Status Ping: <a href="status/ping.html">Here</a>
+		</div>
+
+	</div>
+
+	<h1>Points of Integration</h1>
+
+	<div>
```
Comment (gsilver) [inline src/main/webapp/status.html]: This should be a list.

**Comment by dlhaines** [comment]
Does it make sense for an automated scripts to check this?  That is necessary at least for ping.  These urls should not require authentication.

**Pull request merged**